keyboards/exclusive/e6v2/readme.md

```diff
@@ -13,7 +13,7 @@ The E6-V2 is a 60% keyboard manufactured by Exclusive.
 
 These docs are for the QMK version of the E6-V2 PCB. [More info on qmk.fm](http://qmk.fm/)
```

Comment:
I find this line a bit misleading, considering the BMC PCBs can still run QMK.

Reply:
It is misleading but that is actually what they are called in the GB. I'm trying to keep the wording consistent with what the GB says and what Exclusive has communicated to me.